### PR TITLE
docs: update the slack invite for the openjs foundation

### PR DESCRIPTION
**Pull request by lirantal**
Fix README due to confirming OpenJS Foundation slack here: https://github.com/nodejs/security-wg/issues/812

### CODE REVIEW

**Comment by DanielRuf** [comment]
I guess this is not 100% correct: `[GitHub Logo]`

Also when I click the link it leads me to the landing page of Slack to create a new workspace. Is this invite-only?

`https://app.slack.com/client/T0K2RM7F0/C03Q9MS3KFB` is typically the URL after the login - the one of your client. There should be some address `workspace-name.slack.com`.

**Comment by lirantal** [comment]
Ahh I see. I have a sharable link I can create for the channel but it expires in 14 days so that wouldn't work either. However it should be possible to generate slack invite links.

**Comment by RafaelGSS** [comment]
Yes, I guess we need just to add the channel name and let them enter by themselves. Also, it might work: https://openjs-foundation.slack.com/archives/C03Q1V9M49K

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [![Node.js Security WG](https://img.shields.io/badge/Node.js-Security%20WG-green.svg)]()
 [![Security WG Meetings](https://img.shields.io/badge/YouTube-Security%20WG%20Meetings-red.svg)](  https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ/search?query=Security+WG+meeting)
 [![Security WG Twitter Hashtag](https://img.shields.io/badge/Twitter-%23SecurityWG-blue.svg)](https://twitter.com/search?q=SecurityWG)
-[![GitHub Logo](https://img.shields.io/badge/Join%20us%20on-Slack-e01563.svg)](https://nodejs-security-wg.herokuapp.com/)
+[![GitHub Logo](https://img.shields.io/badge/join%20slack%20on-nodejs--security--wg-green.svg)](https://slack-invite.openjsf.org/)
```
Comment (RafaelGSS) [inline README.md:4]:
```suggestion
[![OpenJS Slack Invite](https://img.shields.io/badge/join%20slack%20on-nodejs--security--wg-green.svg)](https://slack-invite.openjsf.org/)
```

**Comment by lirantal** [comment]
Alrighty then. Good to land or did I miss anything?

**Review by RafaelGSS — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]
LGTM!

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**